source/sass/views/initiatives.scss

```diff
@@ -5,3 +5,62 @@
 .bottom-line {
   border-bottom: 1px solid $light-gray;
 }
+
+// Fellowships--Support
+
+.fellow-support {
+  &-intro {
+    &-help {
+      font-weight: 600;
+    }
+  }
+
+  &-optim {
+    max-width: 45%;
+
+    @media (min-width: $bp-md) {
+      max-width: 85%;
+    }
+  }
+
+  &-logo {
+    display: -webkit-box;
```

Comment:
We have autoprefix set up (see package.json) so for most cases we don't have to add vendor prefix manually each time. `display: flex;` should be good enough!